src/app/shared/pipes/large-number.pipe.ts

```diff
@@ -0,0 +1,8 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({name: 'largeNumber'})
+export class LargeNumberPipe implements PipeTransform {
+  transform(value: number) {
+    return Number(value).toLocaleString(undefined, {useGrouping: true});
```

Comment:
nice! useGrouping is true by default, but it makes it quite clear what this is doing, so 👍